github_actions/run.py

```diff
@@ -176,6 +177,8 @@ def main() -> None:
         _handle_push_event(event)
     elif event.event_name == EVENT_PULL_REQUEST:
         _handle_pr_event(event)
+    elif event.event_name == EVENT_PULL_REQUEST_TARGET:
+        _handle_pr_event(event)
```

Comment:
```suggestion
    elif event.event_name in {EVENT_PULL_REQUEST, EVENT_PULL_REQUEST_TARGET}:
        _handle_pr_event(event)
```

Reply:
Or better yet, declare a constant with the events and use it in the if clause.
For eg:
```python
...
PULL_REQUEST_EVENTS = {EVENT_PULL_REQUEST, EVENT_PULL_REQUEST_TARGET}
...
elif event.event_name in PULL_REQUEST_EVENTS:
    _handle_pr_event(event)
...
```